README.md

```diff
@@ -133,10 +140,10 @@ toybox = { "doll" => 'Sally' }
 ```
 
 - [ ] `toybox.doll`
-- [ ] `toybox['doll']`
-- [ ] `toybox["doll"]`
+- [X] `toybox['doll']`
+- [X] `toybox["doll"]`
 - [ ] `toybox[:doll]`
-- [ ] `toybox.try('doll')`
+- [ ] `toybox.try('doll')` # Hmmmmmmmmmmmmmmmmmmmm feels like this might be a thing
```

Comment:
Haha, you were correct in not selecting this one. 😄 `Object#try` is a rails method and invokes a method on the object so it wouldn't work anyway unless you were using `HashWithIndifferentAccess`.